### DISCUSSION TITLE
Change port for API

### DISCUSSION THREAD

**Discussion by gertyes** [jaredhendrickson13/pfsense-api#298] [Q&A] [answered]
Hy All,

Does anyone know if it is possible to change the listening port for the pfsense API? It is currently on port 80 and I would like to change it to something else.

I need to change the port since it is interfering with some other service connected to my network.

Kind Regards,
Gert.

**Comment by crawc** [comment, marked as answer]
I'm pretty sure it uses the native pfSense webConfigurator port which can be set here: System-> Advanced -> Admin Access -> TCP Port. It is also recommended to run it using HTTPS (SSL/TLS).
